veg2hab/criteria.py

```diff
@@ -150,6 +150,9 @@ def get_opm(self) -> Set[str]:
             )
         }
 
+    def get_format_string(self):
```

Comment:
Ik denk dat het netjes is als de __str__ hierboven ook gewoon zijn eigen get_format_string gebruikt. Dan heb je iets minder dubbelingen in je code staan.